avm/src/daemon/linux.rs

```diff
@@ -118,25 +117,30 @@ pub async fn start(cluster_id: &str, agz_b64: &str, deploy_token: &str) {
   let deploy_token = deploy_token.to_string();
   let agzb64 = agz_b64.to_string();
   task::spawn(async move {
+    // TODO better period determination
+    let period = Duration::from_secs(30);
```

Comment:
So, with respect to re-evaluating which node is in charge, this is fine, and potentially we crank that up to, say, every 1-2 seconds?

Reply:
Based on our conversation, it may be best to crank this from 30 to 180 to be on the safe side for now?